### DISCUSSION TITLE
Readme doesn't say what Mapsforge is

### DISCUSSION THREAD

**Discussion by SomeoneElseOSM** [mapsforge/mapsforge#1353] [Q&A] [answered]
The readme at https://github.com/mapsforge/mapsforge doesn't say anything about what Mapsforge actually is - it just starts with what file format it uses.  The OSM wiki at https://wiki.openstreetmap.org/wiki/Mapsforge does a better job.

**Comment by devemux86** [comment]
GitHub repositories have an About section (top right).

**Comment by SomeoneElseOSM** [comment]
That doesn't really help.  I was trying to write an answer to [this](https://help.openstreetmap.org/questions/85327/wrong-map-style-download-for-offline-use?page=1&focusedAnswerId=85340#85340) OSM help question, and the text "Vector map library and writer - running on Android and Desktop" doesn't really do this.

An example readme that is more informative is https://github.com/gravitystorm/openstreetmap-carto .

**Comment by devemux86** [comment, marked as answer]
Mapsforge is not such a simple one-purpose tool.
It hosts libraries and utilities for developers and map / theme / poi creators.

It is impossible to mention everything on the main page (the about mentions its purpose).
Anyone who is really interested can check the extensive documents (linked on main page).
For map files, render themes, map writer, poi files, poi writer, Android / Desktop libraries, etc.
